caffe2/serialize/inline_container.cc

```diff
@@ -357,7 +371,7 @@ void PyTorchStreamWriter::writeEndOfFile() {
   // Rewrites version info
   std::string version = c10::to_string(version_);
   version.push_back('\n');
-  writeRecord("version", version.c_str(), version.size());
+  writeRecord(version_location_, version.c_str(), version.size());
```

Comment:
We can decide where to write the version based on the maximum version number being produced.

Reply:
`setMinVersion` should control where it gets written.